blog/2024-02-15-private-api-gateways.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Understanding Private API Gateways
```

Comment:
This is the title that was liked best when we discussed how to syndicate it

```suggestion
title: Why are Private API Gateways so hard?
```

Reply:
Sounds good 👍